android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsBrowserState.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import mozilla.components.concept.engine.translate.TranslationError
+
+/**
+ * Value type that represents the state of the translations engine within a [BrowserState].
+ *
+ * @property isEngineSupported The translations engine supports the device architecture.
```

Comment:
```suggestion
 * @property isEngineSupported Whether the translations engine supports the device architecture.
```
Just extra wording to indicate it is a boolean

Reply:
Ty, nice wording